packages/faceted-search/src/dictionary/operator.dictionary.js

```diff
@@ -31,6 +32,13 @@ const standardOperators = t => ({
 		name: 'contains',
 		iconName: 'contains',
 	},
+	[operatorNames.containsIgnoreCase]: {
+		label: t('OPERATOR_CONTAINS_LABEL', {
```

Comment:
why keep same label than for contains?
Newbie question 🔰

Reply:
Checked with PM: "For the user, it's the same", so no need to have a different i18N key 👍

Reply:
Ok thanks 🥇